client/client.go

```diff
@@ -61,7 +89,12 @@ func GetDomainExpiration(hostname string) (domainExpiration time.Duration, err e
 			return domainExpiration, nil
 		}
 	}
-	if whoisResponse, err := whoisClient.QueryAndParse(hostname); err != nil {
+	whoisResponse, err := rdapQuery(hostname)
```

Comment:
Can you add some tests for this change? I've got a couple more requests, but I'd like to start with that